### PR TITLE
test: Removes IDs From Datasource Directory Role Collection Test

### PR DESCRIPTION
**Pull request by andre-dossinger**
## Purpose
* Makes test recording landscape independent

## Does this introduce a breaking change?
```
[ ] Yes
[ x ] No
```

## Pull Request Type

What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ x ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test

* Test the code via automated test

```bash
go test ./...
```

## What to Check

Verify that the following are valid:

* Automated tests are executed successfully

## Checklist for reviewer

The following organizational tasks must be completed before merging this PR:

* [ ] The PR is assigned to the Terraform project and a status is set (typically "in review").
* [ ] The PR has the matching labels assigned to it.
* [ ] The PR has a milestone assigned to it.
* [ ] If the PR closes an issue, the issue is referenced.
* [ ] Possible follow-up items are created and linked.

### CODE REVIEW

**Comment by andre-dossinger** [comment]
Prerequisite for https://github.com/SAP/terraform-provider-btp/issues/59

File: internal/provider/datasource_directory_role_collection_test.go [hunk 1]
```diff
@@ -22,9 +22,10 @@ func TestDataSourceDirectoryRoleCollection(t *testing.T) {
 			ProtoV6ProviderFactories: getProviders(rec.GetDefaultClient()),
 			Steps: []resource.TestStep{
 				{
-					Config: hclProviderFor(user) + hclDatasourceDirectoryRoleCollection("uut", "05368777-4934-41e8-9f3c-6ec5f4d564b9", "Directory Viewer"),
+					Config: hclProviderFor(user) + hclDatasourceDirectoryRoleCollection("uut", "integration-test-dir-se-static", "Directory Viewer"),
+					//Config: hclProviderFor(user) + hclDatasourceDirectoryRoleCollectionByDirectoryId("uut", "c009e317-aa79-40d0-9da8-c9399f066dc1", "Directory Viewer"),
```
Comment (lechnerc77) [inline internal/provider/datasource_directory_role_collection_test.go]: Can we remove that comment?

**Review by lechnerc77 — CHANGES_REQUESTED** [review]

File: internal/provider/datasource_directory_role_collection_test.go (hunk 1 above)
Comment (andre-dossinger) [reply internal/provider/datasource_directory_role_collection_test.go]: The comment is a leftover. Good catch!

**Review by lechnerc77 — APPROVED** [review]
lgtm

**Pull request merged**